midend/simplifyKey.h

```diff
@@ -86,7 +116,9 @@ class DoSimplifyKey : public Transform {
     const IR::Node* postorder(IR::P4Table* table) override;
 };
 
-// Use a policy to decide when a key expression is too complex.
+/**
+ * This pass uses a policy to decide when a key expression is too complex.
```

Comment:
"Determine whether a key expression is too complex"

Also, "too complex" should be defined above.